python/lsst/pipe/tasks/transformMeasurement.py

```diff
@@ -62,22 +73,44 @@ class TransformTask(pipeBase.Task):
     Transformations are defined on a per-measurement-plugin basis. In
     addition, a configurable set of fields may be simply copied from the input
     to the output catalog.
+
+    This task operates on an input SourceCatalog and returns a BaseCatalog
+    containing the transformed results. It requires that the caller supply
+    information on the configuration of the measurement task which produced
+    the input data as well as the world coordinate system and calibration
+    under which the transformation will take place. It provides no
+    functionality for reading or writing data from a Butler: rather,
+    per-dataset-type command line tasks are provided to obtain the appropriate
+    information from a Butler (or elsewhere) and then delegate to this task.
+
+    \section pipe_tasks_transform_initialize Task initialization
+
+    \copydoc \_\_init\_\_
+
+    \section pipe_tasks_transform_invoke Task invocation
+
+    \copydoc run
     """
     ConfigClass = TransformConfig
     _DefaultName = "transform"
 
-    def __init__(self, measConfig, pluginRegistry, inputSchema, *args, **kwargs):
+    def __init__(self, measConfig, pluginRegistry, inputSchema, outputType, *args, **kwargs):
```

Comment:
You can actually get the `PluginRegistry` from `measConfig`: `measConfig.plugins.registry`, so it doesn't need to be a separate argument.

Also, can't comment on a few lines below, since it's not a change, but "measurments" is a typo.